quinn-proto/src/congestion/bbr.rs

```diff
@@ -0,0 +1,672 @@
+use std::fmt::Debug;
+use std::sync::Arc;
+use std::time::{Duration, Instant};
+
+use crate::congestion::bbr_bw_estimation::BandwidthEstimation;
+use crate::congestion::PacketNumber;
+
+use super::{Controller, ControllerFactory};
+use crate::congestion::bbr_min_max::MinMax;
+use crate::connection::paths::RttEstimator;
+
+// Constants based on TCP defaults.
+// The minimum CWND to ensure delayed acks don't reduce bandwidth measurements.
+// Does not inflate the pacing rate.
+const kDefaultMinimumCongestionWindow: u64 = 4 * MAX_DATAGRAM_SIZE;
```

Comment:
Those should be using Rust style, like `const DEFAULT_MINIMUM_CONGESTION_WINDOW`

Reply:
fixed